tests/testthat/example_default_load/dettl.yml

```diff
@@ -15,9 +15,3 @@ load:
   default: TRUE
```

Comment:
Not necessarily for this PR, but I wonder if "automatic" is better than "default" as a name for this

Reply:
Yeah agreed I think that is a better name for what it is doing